capa/features/extractors/dotnetfile.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+import dnfile as dnfile
+
+from capa.features.common import ARCH_I386, ARCH_AMD64, Arch
+from capa.features.extractors.base_extractor import FeatureExtractor
+
+logger = logging.getLogger(__name__)
+
+
+def extract_file_os(**kwargs):
+    # TODO yield OS(OS_ANY), 0x0?
+    yield
+
+
+def extract_file_arch(pe, **kwargs):
+    # TODO also see 32 - Assembly Table?
+    if pe.net.Flags.CLR_32BITREQUIRED:
+        yield Arch(ARCH_I386), 0x0
+    else:
+        yield Arch(ARCH_AMD64), 0x0
+
+
+def extract_global_features(pe, buf):
+    """
+    extract global features from given workspace
+
+    args:
+      pe (pefile.PE): the parsed PE
+      buf: the raw sample bytes
+
+    yields:
+      Tuple[Feature, VA]: a feature and its location.
+    """
+    for handler in GLOBAL_HANDLERS:
+        for feature, va in handler(pe=pe, buf=buf):
+            yield feature, va
+
+
+GLOBAL_HANDLERS = (
+    # TODO extract_file_os,
+    extract_file_arch,
+)
+
+
+class DotnetfileFeatureExtractor(FeatureExtractor):
```

Comment:
```suggestion
class DnfileFeatureExtractor(FeatureExtractor):
```

??? up for discussion